external-providers/golang-dependency-provider/main.go

```diff
@@ -17,42 +19,47 @@ const (
 )
 
 // TODO implement this for real
-func (g *golangServiceClient) findGoMod() string {
-	var depPath string
-	if g.config.DependencyPath == "" {
-		depPath = "go.mod"
-	} else {
-		depPath = g.config.DependencyPath
-	}
-	f, err := filepath.Abs(filepath.Join(g.config.Location, depPath))
+func main() {
+	ll, err := GetDependenciesDAG()
 	if err != nil {
-		return ""
-	}
-	return f
-}
-
-func (g *golangServiceClient) GetDependencies() (map[uri.URI][]*provider.Dep, error) {
-	ll, err := g.GetDependenciesDAG()
-	if err != nil {
-		return nil, err
+		log.Fatal(err)
+		return
 	}
 	if len(ll) == 0 {
-		return nil, nil
+		log.Fatal(fmt.Errorf("no dependencies"))
```

Comment:
I don't think we want to error out, sometimes there may be no dependencies

Reply:
that was left out from when i was trying to debug, i was checking if i was getting any output from the binary